pom.xml

```diff
@@ -130,10 +130,42 @@
           </execution>
         </executions>
       </plugin>
+        <plugin>
+            <groupId>org.openrewrite.maven</groupId>
+            <artifactId>rewrite-maven-plugin</artifactId>
+      </plugin>
     </plugins>
     <pluginManagement>
       <plugins>
-        <plugin>
+          <plugin>
+              <groupId>org.openrewrite.maven</groupId>
+              <artifactId>rewrite-maven-plugin</artifactId>
+              <version>6.15.0</version>
+              <configuration>
+                  <activeRecipes>
+                      <recipe>org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods</recipe>
```

Comment:
please lets avoid any further to fix the **broken window**. Now its free and the best moment, everything else will be much more expensive, as the bugs and flaws will not be ignored by anybody.

Some actually like to invest the effort, fixing things up.